articles/server-apis/nodejs.md

```diff
@@ -19,6 +19,14 @@ snippets:
 
 ## NodeJS API Tutorial
 
+You can get started by either downloading the seed project or if you would like to add Auth0 to an existing application you can follow the tutorial steps.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* NodeJS 4.2
```

Comment:
@ntotten Hi! I added express already, can you see it? do you want me to add anything else?